### PR TITLE
update docs for Subnet Configuration Generator v1

### PR DESCRIPTION
**Pull request by wanwiset25**

### CODE REVIEW

File: .github/workflows/docusaurus-deploy-develop.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 on:
   push:
     branches:
-      - main 
+      - develop
```
Comment (liam-icheng-lai) [inline .github/workflows/docusaurus-deploy-develop.yml:4]: main?

File: docs/.DS_Store
Comment (benjamin202410) [inline docs/.DS_Store:1]: gitignore

File: docs/deployment/1_launch_subnet.md [hunk 2]
```diff
@@ -5,58 +5,70 @@ sidebar_position: 1
...
-  2. Pull the `generator.sh` script from the generator Github repo
+## Video Walkthrough
+<iframe width="720" height="400" src="https://www.youtube.com/embed/m-sPbMrB8ow" frameborder="0" allow="fullscreen"></iframe>
+
+## Generate Subnet Configs With UI
+
+  1. Pull `generator.sh` script from the generator Github repo
   ```
-  curl -O https://raw.githubusercontent.com/XinFinOrg/XinFin-Node/master/subnet/deployment-generator/script/generate.sh
+  curl -O https://raw.githubusercontent.com/XinFinOrg/XinFin-Node/generator-ui-concept/subnet/deployment-generator/scripts/generate.sh
```
Comment (benjamin202410) [inline docs/deployment/1_launch_subnet.md]: master or you will change it later?

File: docs/usage/_category_.json [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
     "type": "generated-index",
-    "description": "The guide for XDC Subnet user interface"
+    "description": "The guide to using XDC-Subnet"
   }
-}
\ No newline at end of file
+}
+
+
+
```
Comment (benjamin202410) [inline docs/usage/_category_.json]: too many new lines

**Review by benjamin202410 — APPROVED** [review]
Overall is good to me

**Review by liam-lai — APPROVED** [review]

File: docs/deployment/1_launch_subnet.md (hunk 2 above)
Comment (wanwiset25) [reply docs/deployment/1_launch_subnet.md]: this is convenient for testing will change before merge

**Pull request merged**